packages/core/src/model.d.ts

```diff
@@ -220,15 +224,15 @@ export type WhereOptions<TAttributes = any> = AllowNotOrAndWithImplicitAndArrayR
 
 // number is always allowed because -Infinity & +Infinity are valid
 /**
- * This type represents a valid input when describing a {@link RANGE}.
+ * This type represents a valid input when describing a {@link DataTypes.RANGE}.
```

Comment:
Also holds for the Range type below, but do we want to link to `DataTypes.RANGE` or `<internal>.RANGE`?